.pipelines/templates/PackageBuild.yml

```diff
@@ -160,6 +160,12 @@ steps:
         displayName: "Populate RPMs cache"
 
   - script: |
+      toolchain_archive="$(find "$(Pipeline.Workspace)" -name "${{ parameters.customToolchainTarballName }}" -print -quit)"
+      if [[ ! -f "$toolchain_archive" ]]; then
```

Comment:
There is no guarantee the user will pass any custom toolchain archives, so we can't expect it to be present.